CHANGELOG.md

```diff
@@ -1,15 +1,24 @@
+## <small>1.0.1 (2021-09-16)</small>
+
+* 1.0.1 ([686b260](https://github.com/ekino/veggies/commit/686b260))
```

Comment:
wrong message / doesn't match the commit or its link.

Also, this isn't really useful IMO, we can omit it from the changelog

Reply:
You're right it's probably a mistake during manual release process when I used the update changelog command.

I'will remove this line.